src/main/java/github/scarsz/discordsrv/DiscordSRV.java

```diff
@@ -1739,27 +1739,29 @@ public void givePlayerOnlineRole(Player player) {
         if (player == null) throw new IllegalArgumentException("player cannot be null");
 
         // retrieves the role if possible, if not returns null
-        String roleName = DiscordSRV.config().dget("OnlineRoleName").asString();
+        String roleName = config().getString("OnlineRoleName");
         Role role = DiscordUtil.getRoleByName(getMainGuild(), roleName);
-        if (role == null){
-            getLogger().severe("Role \""+roleName+"\" could not be found. Make sure OnlineRoleName is correctly set in the config.yml");
+        if (role == null) {
+            warning("Role \"" + roleName + "\" could not be found. Make sure OnlineRoleName is correctly set in the config.yml");
             return;
         }
 
         // retrieves the discord id, if possible. If not, returns null
         String discordId = accountLinkManager.getDiscordId(player.getUniqueId());
+
         if (discordId == null) {
-            getLogger().severe("Online role is enabled, but discord linking is not.");
+            warning("Online role feature is enabled but no discord account has been linked");
```

Comment:
These should actually be debug messages considering these will be reached on a normal basis